testing/upload_validation.ts

```diff
@@ -715,6 +745,22 @@ ${endpointKey ? 'endpointKey is set' : `requiresEndpointUrl is ${requiresEndpoin
     .filter(([authType]) => authType in variousSupportedAuthenticationTypes)
     .map(([_authType, schema]) => schema);
 
+  const reservedAuthenticationNames = Object.values(ReservedAuthenticationNames).map(value => value.toString());
```

Comment:
Interesting, how come `.toString()` was needed?

Reply:
Because this :/
![Screenshot 2024-10-08 at 12 54 24 PM](https://github.com/user-attachments/assets/1d963a11-6f85-4b9c-96bc-86d8cbad6b70)

Reply:
Ah yes, the classic reason we have `array.includes()` on the `coda` side.